nemo/collections/tts/data/tts_dataset.py

```diff
@@ -185,7 +185,6 @@ def __init__(
         self.phoneme_probability = None
         if isinstance(self.text_tokenizer, BaseTokenizer):
             self.text_tokenizer_pad_id = text_tokenizer.pad
-            self.tokens = text_tokenizer.tokens
```

Comment:
this was not used anywhere, so that's why you remove it. Is my understanding correct?

Reply:
Yes, I was not able to find any references to this and removed it. Test runs seem fine without it as well.